src/TestFramework/PhpUnit/Coverage/CoverageXmlParser.php

```diff
@@ -134,7 +134,8 @@ private function getSourceFilePath(\DOMXPath $xPath, string $relativeCoverageFil
             return $realPath;
         }
 
-        throw new \Exception('Source file was not found');
+        throw new \Exception(
```

Comment:
can we create smth like CoverageException? or put this message into CoverageDoesNotExistException?